example/android/.project

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>android</name>
+	<comment>Project android created by Buildship.</comment>
```

Comment:
`by Tozny`

Reply:
I believe these are auto generated gradle files... but from googling they seem to be part of Eclipse settings which i don't have ❓